spec/integration/admin_view_spec.rb

```diff
@@ -4,14 +4,16 @@
   include Capybara::DSL
 
   before(:each) do
```

Comment:
before each, mike? and then going to a new route in the first test? (yeah, i know i wrote this test... :8ball: ). should we strip the before out before pushing? (ben said make comments in question form right? :snowboarder: :last_quarter_moon: